Submitted by Vladimir B. Grebenschikov this patch recognizes Samsung's
images as JPEGs.

diff --git utils.c utils.c
index 36ff168..5a7ff87 100644
--- utils.c
+++ utils.c
@@ -324,7 +324,7 @@ is_audio(const char * file)
 int
 is_image(const char * file)
 {
-	return (ends_with(file, ".jpg") || ends_with(file, ".jpeg"));
+	return (ends_with(file, ".jpg") || ends_with(file, ".jpeg") || ends_with(file, ".mpo"));
 }
 
 int
